hydra-node/test/Hydra/Chain/Direct/WalletSpec.hs

```diff
@@ -196,6 +198,24 @@ prop_seenInputsAreConsumed =
 -- coverFee
 --
 
+prop_setsMinUTxOValue :: Property
+prop_setsMinUTxOValue =
+  forAllBlind (resize 0 genLedgerTx) $ \tx ->
+    forAllBlind (reasonablySized $ genOutputsForInputs tx) $ \lookupUTxO ->
+      forAllBlind (reasonablySized genUTxO) $ \walletUTxO ->
+        -- Generate another txOut and make it deliberately "under-valued"
+        forAll arbitrary $ \txOut -> do
+          let txOutWithoutADA = txOut & coinTxOutL .~ mempty
```

Comment:
Suggestion: replace `arbitrary` with `genTxOutWithoutAda` and define the latter as a helper function